src/util.ts

```diff
@@ -250,8 +250,8 @@ function querySideloadingRegistry(commands: string[]): Promise<string> {
   });
 }
 
-function addManifestToRegistry(manifestPath: string): Promise<any> {
-  return querySideloadingRegistry(['Set-ItemProperty -LiteralPath $RegistryPath -Name "' + manifestPath + '" -Value "' + manifestPath + '"']);
+function addManifestToRegistry(parsedGuid: string, manifestPath: string): Promise<any> {
+  return querySideloadingRegistry(['Set-ItemProperty -LiteralPath $RegistryPath -Name "' + parsedGuid + '" -Value "' + manifestPath + '"']);
```

Comment:
This isn't directly part of your change, but it would be nice to use the TypeScript back-tick string to do replacements instead of concatenating the strings. 

The manifest.xml is for multiple hosts, and there is also manifest.{host}.xml which has a subset of it for each host. If you want to target only a single host, run “npm run convert-to-single-host {host}” e.g. “npm run convert-to-single-host excel”. 
```
return querySideloadingRegistry([`Set-ItemProperty -LiteralPath $RegistryPath -Name "${manifestId}" -Value "${manifestPath}"`]);
```

Reply:
I plan not to do the TypeScript back-tick string change in this change. To keep it simple.

Reply:
Fine.